src/central_control/scripts/central_control.py

```diff
@@ -36,8 +36,10 @@ def handle_steering_data(self, steering_data):
     def handle_breaking_data(self, braking_data):
         print("Obtained braking data")
 
-    def handle_throttling_data(self, throttling_data):
+    def handle_throttling_data(self, throttling_data: Float64):
```

Comment:
Just curious any reason for float64? Would our throttle data ever be that complex?

Reply:
The throttle value for airsim uses a float, only other option I know of is Float32. Not sure if that would provide any real improvement though. Was there a data type you were thinking of?